rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/parsetreetoast/bif.kt

```diff
@@ -25,10 +25,18 @@ internal fun RpgParser.BifContext.toAst(conf: ToAstConfiguration = ToAstConfigur
         this.bif_abs() != null -> this.bif_abs().toAst(conf)
         this.bif_int() != null -> this.bif_int().toAst(conf)
         this.bif_editw() != null -> this.bif_editw().toAst(conf)
+        this.bif_rem() != null -> this.bif_rem().toAst(conf)
         else -> TODO(this.text + " " + toPosition(conf.considerPosition))
     }
 }
 
+internal fun RpgParser.Bif_remContext.toAst(conf: ToAstConfiguration): Expression {
+    return RemExpr(
+        this.expression(0).toAst(conf),
```

Comment:
Normally I would suggest adding labels in the parser grammar, so we can use them here. It makes the code more readable and less error prone

Reply:
Sorry, I don't understand this: could you give me more explanations? Thanks.

Reply:
Sure. In the grammar you can have rule such has:

```
remExpr: expression REM expression ;
```
In this case you can access the two expressions only using an index, as you did.

But you can add labels:

```
remExpr: left=expression REM right-expression ;
```

Now you can access the single expressions using their label name.